model-mesh/README.md

```diff
@@ -32,3 +32,41 @@ Example ServingRuntime and Predictors can be found at:  https://github.com/kserv
 ### Overlays
 
 None
+
+### Installation process
+
+Following are the steps to install Model Mesh as a part of OpenDataHub install:
+
+1. Install the OpenDataHub operator
+2. Create a KfDef that includes the model-mesh component with the odh-model-controller overlay.
+   
+```
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  name: opendatahub
+  namespace: opendatahub
+spec:
+  applications:
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: odh-common
+      name: odh-common
+    - kustomizeConfig:
+        overlays:
+          - odh-model-controller
+        repoRef:
+          name: manifests
+          path: model-mesh
+      name: model-mesh
+  repos:
+    - name: kf-manifests
+      uri: https://github.com/opendatahub-io/manifests/tarball/v1.4.0-rc.2-openshift
+    - name: manifests
+      uri: https://api.github.com/repos/opendatahub-io/odh-manifests/tarball/pull/599/head
```

Comment:
This uri doesn't seem good to me.

Reply:
Yes, that uri should be pointed at master.......I think this was a copy/paste from my testing instructions from the PR that was to add this functionality....which has since been merged.

Reply:
Done, took the `kf-manifests` and the corresponding uri out.